atomics/T1145/T1145.yaml

```diff
@@ -9,14 +9,21 @@ atomic_tests:
     File extensions include: .key, .pgp, .gpg, .ppk., .p12, .pem, pfx, .cer, .p7b, .asc
   supported_platforms:
     - windows
+
+  dependency_executor_name: powershell
+  dependencies:
+  - description: |
+      Create a key file to ensure that the findstr function gets at least one result.
+    prereq_command: |
+      if (Test-Path $env:TEMP\cert.key) { exit 0 } else { exit 1 }
+    get_prereq_command: |
+      New-Item -Path $env:TEMP\cert.key -Force | Out-Null
+
```

Comment:
The test is to search recursively for key files. I think the idea is for the detection to fire if a search is done for key files regardless of whether one is found. In this case, I think we can do away with actually having to create one for the test to find. I would just remove this prereq and leave off the "echo "ATOMICREDTEAM" > %windir%\cert.key" line you already removed.

Reply:
@clr2of8 got your requests fixed